Marlin/Marlin_main.cpp

```diff
@@ -4829,6 +4844,66 @@ inline void gcode_M31() {
 
 #endif // SDSUPPORT
 
+#if ENABLED(PISTON_EXTRUDER)
+  inline void gcode_M35() { //turn on dryer fan
```

Comment:
You can make such changes within the same pull request, just by adding commits to your branch. At this point, however, it will probably be easier to start over.